dashel/dashel-win32.cpp

```diff
@@ -1440,18 +1443,19 @@ namespace Dashel
 	{
 		while(step(-1));
 	}
-	
+
 	bool Hub::step(const int timeout)
 	{
-		HANDLE hEvs[64] = { hTerminate };
-		WaitableStream *strs[64] = { NULL };
-		EvType ets[64] = { EvClosed };
-		
+		lock();
+
+		std::vector<HANDLE> hEvs(streams.size());
+		std::vector<EvType> ets(streams.size());
+		std::vector<WaitableStream*> strs(streams.size());
+
 		// Wait on all our events.
 		DWORD ms = timeout >= 0 ? timeout : INFINITE;
-		
+
```

Comment:
buggy indentation

Reply:
I was buggy before !

Reply:
In Dashel (and Aseba), the current style is to keep indentation on empty lines. I know that currently this style is out of fashion, but then it was common. Also, I find the out-of-fashionableness somewhat arbitrary.